### PR TITLE
Bump cabal version to 1.8 and update CHANGELOG

### PR DESCRIPTION
**Pull request by larskuhtz**
## TODO

* [x] Fill in new kill switch date in CHANGELOG.md
* [x] Fill in release data in CHANGELOG.md

### CODE REVIEW

**Comment by slpopejoy** [comment]
We should consider fixing the comment at https://github.com/kadena-io/chainweb-node/pull/974/files#diff-562725994970d8cbbeb9cded97cd4fc9R763 or not having a comment at all

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
-## Unreleased
+## 1.8 (TODO)
 
 This version replaces all previous versions. Any prior version will stop working
 on **2020-04-30T00:00:00Z**. Node administrators must upgrade to this version
 before that date.
 
+This version will stop working on **2020-06-11T00:00:00Z**.
+
```
Comment (slpopejoy) [inline CHANGELOG.md:10]: Please add #1019, #1004, #1002

File: node/ChainwebNode.hs [hunk 2]
```diff
@@ -469,7 +469,7 @@ pkgInfoScopes =
 -- KILLSWITCH for version 1.7
 --
 killSwitchDate :: Maybe String
-killSwitchDate = Just "2020-04-30T00:00:00Z"
+killSwitchDate = Just "2020-06-11T00:00:00Z"
```
Comment (slpopejoy) [inline node/ChainwebNode.hs:472]: 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,21 @@
 # `chainweb-node` Changelog
 
-## Unreleased
+## 1.8 (TODO)
```
Comment (slpopejoy) [inline CHANGELOG.md:3]: We should just set this to like "a day before the kill switch" (2020-04-29) or something

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,17 +1,21 @@
...
 
-*   The option to persist the peer database is removed (TODO). The functionality
+*   The option to persist the peer database is removed. The functionality
     was rarely used and can be simulated by using the peer REST API endpoints
-    along with the know peers configuration option.
+    along with the know peers configuration option. (#1010)
+
+*   Removed the PUT endpoints from the `/chain/header` APIs. (#1002).
 
 ## 1.7 (2019-03-26)
```
Comment (slpopejoy) [inline CHANGELOG.md:20]:
```suggestion
## 1.7 (2020-03-26)
```

File: CHANGELOG.md (hunk 1 above)
Comment (larskuhtz) [reply CHANGELOG.md:10]: 1002 is already mentioned below. I think 1004 is not required since it's not a user visible change.

Comment (mightybyte) [reply CHANGELOG.md:10]: Added

File: CHANGELOG.md (hunk 3 above)
Comment (mightybyte) [reply CHANGELOG.md]: I set it to the planned release date.

File: CHANGELOG.md (hunk 4 above)
Comment (mightybyte) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,19 +1,27 @@
...
+*   The option to persist the peer database is removed. The functionality
     was rarely used and can be simulated by using the peer REST API endpoints
-    along with the know peers configuration option.
+    along with the know peers configuration option. (#1010)
+
+*   Removed the PUT endpoints from the `/chain/header` APIs. (#1002).
+
+*   Make block header validation aware of adjacent parents. (#1004).
+
+*   Increased the default block gas limit by 10x to 150,000. (#1019).
```
Comment (larskuhtz) [inline CHANGELOG.md:22]: We should also mention that node maintainers would have to adjust possible settings of this value in the configuration, for this change to become effective.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,19 +1,27 @@
...
-*   The option to persist the peer database is removed (TODO). The functionality
+*   The option to persist the peer database is removed. The functionality
     was rarely used and can be simulated by using the peer REST API endpoints
-    along with the know peers configuration option.
+    along with the know peers configuration option. (#1010)
+
+*   Removed the PUT endpoints from the `/chain/header` APIs. (#1002).
+
+*   Make block header validation aware of adjacent parents. (#1004).
+
```
Comment (larskuhtz) [inline CHANGELOG.md:21]: This change doesn't affect users. It's a noop at this point.

```suggestion
```

File: CHANGELOG.md (hunk 5 above)
Comment (mightybyte) [reply CHANGELOG.md]: Good point.  Updated.

**Pull request merged**